package.json

```diff
@@ -16,23 +19,23 @@
     "serve": "http-server docs"
   },
   "dependencies": {
-    "@types/assertion-error": "1.0.30",
-    "@types/chai": "3.5.2",
-    "@types/classnames": "0.0.31",
-    "@types/dom4": "1.5.20",
-    "@types/enzyme": "2.8.0",
-    "@types/mocha": "2.2.32",
-    "@types/pure-render-decorator": "0.2.27",
-    "@types/react": "0.14.40",
-    "@types/react-addons-css-transition-group": "0.14.17",
-    "@types/react-addons-transition-group": "0.14.17",
-    "@types/react-dom": "15.5.0",
-    "@types/tether": "1.1.27",
-    "autoprefixer": "^7.1.2",
+    "@types/assertion-error": "^1.0.30",
+    "@types/chai": "^4.0.4",
+    "@types/classnames": "^2.2.3",
+    "@types/dom4": "^1.5.20",
+    "@types/enzyme": "^2.8.0",
+    "@types/mocha": "^2.2.43",
+    "@types/pure-render-decorator": "^0.2.28",
+    "@types/react": "^16.0.14",
+    "@types/react-addons-css-transition-group": "^15.0.3",
+    "@types/react-addons-transition-group": "^15.0.1",
+    "@types/react-dom": "^16.0.1",
```

Comment:
🎆 awesome sauce! although might it be more accurate to use the latest 15.x types instead?

Reply:
The types are basically the same and it's much easier to upgrade everything to latest; we won't be able to use new features at runtime anyway since we run react 15 in development.

Reply:
definitely buy the upgrade/similarity arguments, but my concern with this change is that new features are available at compile time but not at run time.

if you're not concerned then I'm chill.